app/config/index.js

```diff
@@ -1,15 +1,82 @@
+const Joi = require('joi')
+
 if (process.env.NODE_ENV !== 'test') {
   require('dotenv').config()
 }
 
+const schema = Joi.object({
+  env: Joi.string().valid('development', 'test', 'production').required(),
+
+  fundingFarmingApiUri: Joi.string().uri().required(),
+  fundingFarmingApiKey: Joi.string().required(),
```

Comment:
This one can be removed too

Reply:
It seems to be used in 2 places actually. Still remove it?

![Screenshot 2024-06-24 at 14 34 48](https://github.com/DEFRA/fcp-find-ai-frontend/assets/86333618/412f626b-2477-4b12-b3a2-1ad94beade69)

Reply:
Yes, if you can remove the usages too. We don't use this environment variable after switching to langchain. FARMING_FUNDING_API_URI can be removed too

Reply:
done